01-Basic/01-variables.js

```
@@ -37,4 +37,144 @@ var persona = {
+{
+    console.log('Valor dentro del bloque', age)
+}
+
+console.log('Valor fuera del boque', age)
+age = age * 2 
+console.log('valor modification', age)
+
+
+
+// JavaScript
+let name = 'Miguel'
```

Biome (1.9.4)

[error] Shouldn't redeclare 'name'. Consider to delete it or rename it.

'name' is defined here:


(lint/suspicious/noRedeclare)